docker/app/Dockerfile

```diff
@@ -8,18 +8,19 @@ RUN apk add --update --no-cache  \
   build-base \
   postgresql-dev \
   postgresql-client \
-  tzdata
+  tzdata \
+  yarn
 
 # Get bundler 2.0
 RUN gem install bundler
 
-RUN mkdir /app
```

Comment:
No need to run `mkdir /app` before `WORKDIR` as WORKDIR will create the directory if it needs creating.